ipatests/prci_definitions/nightly_latest.yaml

```diff
@@ -2125,11 +2125,23 @@ jobs:
       class: RunPytest
       args:
         build_url: '{fedora-latest/build_url}'
-        test_suite: test_integration/test_ipa_ipa_migration.py
+        test_suite: test_integration/test_ipa_ipa_migration.py::TestIPAMigrateCLIOptions test_integration/test_ipa_ipa_migration.py::TestIPAMigrationStageMode test_integration/test_ipa_ipa_migration.py::TestIPAMigrationProdMode
```

Comment:
You may need to increase the timeout for this test. Use 9000 here and 7200 for the AD one, and use the same values on all the yaml files.